lib/dinosaur.js

```diff
@@ -22,19 +22,17 @@ function Dinosaur(canvas, dino_img_left, dino_img_right) {
 Dinosaur.prototype.reborn = function() {
   this.x = this.canvas.width / 2;
   this.y = this.canvas.height - this.height - 10;
```

Comment:
I'm not sure how we handle the hard coding of the 10. This may just be one of those things we hard code or have a default value somewhere.